packages/vscode-extension/src/common/License.ts

```diff
@@ -112,51 +78,45 @@ export enum Feature {
 
 // #endregion Features
 
-// #region Feature By License
-
-export const FreeFeatures: Set<Feature> = new Set<Feature>([
-  Feature.AndroidSmartphoneEmulators,
-  Feature.AppSwitcherButton,
-  Feature.ComponentPreview,
-  Feature.Debugger,
-  Feature.DeviceAppearanceSettings,
-  Feature.DeviceFontSizeSettings,
-  Feature.ElementInspector,
-  Feature.ExpoRouterIntegration,
-  Feature.HomeButton,
-  Feature.IOSSmartphoneSimulators,
-  Feature.JSLogging,
-  Feature.JSProfiler,
-  Feature.NetworkInspection,
-  Feature.OpenDeepLink,
-  Feature.OutlineRenders,
-  Feature.ReactProfiler,
-  Feature.ReactQueryDevTools,
-  Feature.RadonConnect,
-  Feature.ReduxDevTools,
-  Feature.VolumeButtons,
-]);
-
-export const ProFeatures: Set<Feature> = new Set<Feature>([
-  Feature.AndroidTabletEmulators,
-  Feature.AndroidPhysicalDevice,
-  Feature.Biometrics,
-  Feature.DeviceLocalizationSettings,
-  Feature.DeviceRotation,
-  Feature.IOSTabletSimulators,
-  Feature.LocationSimulation,
-  Feature.Permissions,
-  Feature.RadonAI,
-  Feature.ScreenRecording,
-  Feature.ScreenReplay,
-  Feature.Screenshot,
-  Feature.SendFile,
-  Feature.StorybookIntegration,
-  ...FreeFeatures,
-]);
-
-export const TeamFeatures: Set<Feature> = new Set<Feature>([...ProFeatures]);
-
-export const EnterpriseFeatures: Set<Feature> = new Set<Feature>([...TeamFeatures]);
+// #region Default Features Availability
+
+export const DefaultFeaturesAvailability = {
+  [Feature.AndroidSmartphoneEmulators]: FeatureAvailabilityStatus.AVAILABLE,
+  [Feature.AndroidTabletEmulators]: FeatureAvailabilityStatus.PAYWALLED,
+  [Feature.AndroidPhysicalDevice]: FeatureAvailabilityStatus.PAYWALLED,
+  [Feature.AppSwitcherButton]: FeatureAvailabilityStatus.AVAILABLE,
+  [Feature.Biometrics]: FeatureAvailabilityStatus.PAYWALLED,
+  [Feature.ComponentPreview]: FeatureAvailabilityStatus.AVAILABLE,
+  [Feature.Debugger]: FeatureAvailabilityStatus.AVAILABLE,
+  [Feature.DeviceAppearanceSettings]: FeatureAvailabilityStatus.AVAILABLE,
+  [Feature.DeviceFontSizeSettings]: FeatureAvailabilityStatus.AVAILABLE,
+  [Feature.DeviceLocalizationSettings]: FeatureAvailabilityStatus.PAYWALLED,
+  // FRYTKI - this is changed only for testing
+  [Feature.DeviceRotation]: FeatureAvailabilityStatus.ADMIN_DISABLED,
```

Comment:
Unfrytkify this before merging

```suggestion
  [Feature.DeviceRotation]: FeatureAvailabilityStatus.PAYWALLED,
```